src/endpoints/index.js

```diff
@@ -1,12 +1,14 @@
 const charon = require("./charon");
 const cli = require("./cli");
+const groups = require("./groups");
 const sources = require("./sources");
 const static_ = require("./static");
 const users = require("./users");
 
 module.exports = {
   charon,
   cli,
+  groups,
```

Comment:
Changes in this file should come with 77edeb6c7024c91de4f2d89b291a43b1814ec103, not f2b1eaffad45ba93bf54fe8d360edf74b3719892, otherwise this doesn't work:

https://github.com/nextstrain/nextstrain.org/blob/77edeb6c7024c91de4f2d89b291a43b1814ec103/src/app.js#L303-L304

Reply:
Whoops. Thanks!